contracts/aggregator/arbitrum/flashloan/helpers.sol

```diff
@@ -14,6 +14,23 @@ import {
 contract Helper is Variables {
     using SafeERC20 for IERC20;
 
+     /**
+     * @dev Approves the token to the spender address with allowance amount.
+     * @notice Approves the token to the spender address with allowance amount.
+     * @param token_ token for which allowance is to be given.
+     * @param spender_ the address to which the allowance is to be given.
+     * @param amount_ amount of token.
+    */
+    function approve(address token_, address spender_, uint256 amount_) internal {
+        TokenInterface tokenContract_ = TokenInterface(token_);
+        try tokenContract_.approve(spender_, amount_) {
+
+        } catch {
+            tokenContract_.approve(spender_, 0);
+            tokenContract_.approve(spender_, amount_);
```

Comment:
Why can't we do `safeApprove` just on these 2 lines?

Reply:
Done!